### ISSUE TITLE
Not able to run the code

### ISSUE THREAD

**Issue by GirishKumarSharma** [MuhammadHurKhan/Stock-Price-Prediction-Model#1]
`girish@mypc:~$ git clone https://github.com/MuhammadHurKhan/Stock-Price-Prediction-Model`
`Cloning into 'Stock-Price-Prediction-Model'...`
`remote: Enumerating objects: 9, done.`
`remote: Counting objects: 100% (9/9), done.`
`remote: Compressing objects: 100% (7/7), done.`
`remote: Total 9 (delta 0), reused 0 (delta 0), pack-reused 0`
`Receiving objects: 100% (9/9), done.`
`girish@mypc:~$ cd Stock-Price-Prediction-Model/`
`girish@mypc:~/Stock-Price-Prediction-Model$ ls`
` README.md  'Stock Price Prediction.py'`
`girish@mypc:~/Stock-Price-Prediction-Model$ python3 Stock\ Price\ Prediction.py `
`2023-03-01 21:06:07.742261: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library `'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory`
`2023-03-01 21:06:07.742280: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not `have a GPU set up on your machine.`
`Enter the ticker symbol of the stock you want to predict: ITC.BO`
`[*********************100%***********************]  1 of 1 completed`
`2023-03-01 21:06:17.809280: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library` 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory`
`2023-03-01 21:06:17.809318: W tensorflow/stream_executor/cuda/cuda_driver.cc:269] failed call to cuInit: UNKNOWN ERROR` `(303)`
`2023-03-01 21:06:17.809340: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be` `running on this host (mypc): /proc/driver/nvidia/version does not exist`
`2023-03-01 21:06:17.809500: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with` `oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2` `FMA`
`To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.`
`6411/6411 [==============================] - 95s 15ms/step - loss: 5.8576e-04`
`23/23 [==============================] - 1s 8ms/step`
`/home/girish/Stock-Price-Prediction-Model/Stock Price Prediction.py:92: SettingWithCopyWarning: `
`A value is trying to be set on a copy of a slice from a DataFrame.`
`Try using .loc[row_indexer,col_indexer] = value instead`
``
`See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning`-`view-versus-a-copy`
`  valid['Predictions'] = predictions`
`girish@mypc:~/Stock-Price-Prediction-Model$ `
